src/compose/network-manager.ts

```diff
@@ -70,7 +70,7 @@ export async function remove(network: Network) {
 }
 
 export async function supervisorNetworkReady(): Promise<boolean> {
-	const networkExists = exists(
+	const networkExists = await exists(
```

Comment:
This will also get caught automatically in typescript 4.3: https://devblogs.microsoft.com/typescript/announcing-typescript-4-3-beta/#errors-on-always-truthy-promise-checks 🎉

Reply:
Awesome! I was sure there is a way to detect this sort of issue..

Reply:
@cywang117 look